yang/frr-nexthop.yang

```diff
@@ -188,6 +188,17 @@ module frr-nexthop {
         "Nexthop is directly connected.";
     }
 
+    leaf srte-color {
+      when "../nh-type = 'ip4' or
+            ../nh-type = 'ip6' or
+            ../nh-type = 'ip4-ifindex' or
+            ../nh-type = 'ip6-ifindex'";
+      type uint32;
+      default "0";
```

Comment:
It's probably better to remove this default, since color 0 should be different than having no color at all (although in the code we use zero to represent the absence of a value for now). This shouldn't impact the code in any way since this leaf will have a `destroy()` callback anyway because of its "when" statement.